articles/group-operator/group.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/group-operator-in-mongodb/
+title: Group operator in MongoDB
+description:  Group operator is a very important operator in MongoDB language and it is a part of aggregation in MongoDB so this article will cover $group operator, its applications along with examples.
+author: harit-joshi
+date: 21:52:42 2021 +0530
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/group-operator/hero.jpg
+    alt: Group operator in MongoDB Image
+---
+Group operator is a very important operator in MongoDB language and it is a part of aggregation in MongoDB.
```

Comment:
Need a better into on Group operator. Also, discuss What is MongoDB shortly in the intro.